app/lib/features/tasks/widgets/task_item.dart

```diff
@@ -193,13 +212,11 @@ class TaskItem extends ConsumerWidget {
     );
   }
 
-  Key doneKey() {
-    final taskId = task.eventIdStr();
+  Key doneKey(String taskId) {
```

Comment:
these are not necessary. `taskId` is an attribute of the widget, we don't have to pass it in.

```suggestion
  Key doneKey() {
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2046/commits/633f3a707a97476d9b2e667a0df11395dd3825cc